hnn_core/network.py

```diff
@@ -500,7 +568,7 @@ def write(self, fname):
         for trial_idx in range(len(self._times)):
             with open(str(fname) % (trial_idx,), 'w') as f:
                 for spike_idx in range(len(self._times[trial_idx])):
-                    f.write('{:.3f}\t{}\t{}\n'.format(
+                    f.write('{:.3f}\t{}\t{}\t\n'.format(
```

Comment:
was this intentional?

Reply:
Mistake carried over from previous commits that changed the write function. It just looks weird because of the rebase.